ui/app/styles/components/popup-menu.scss

```diff
@@ -135,6 +135,8 @@
 }
 
 .ember-basic-dropdown-content {
+  background-color: transparent;
```

Comment:
This breaks some styling for the namespace-picker, but we can probably add a background for it:
![Screen Shot 2019-08-19 at 10 37 54 AM](https://user-images.githubusercontent.com/39469/63278911-c858d700-c26d-11e9-8f0f-0ffe041c6168.png)

Reply:
oh, thank you for catching this!